test/Conversion/StandardToHandshake/test1_remove_block.mlir

```diff
@@ -0,0 +1,60 @@
+// NOTE: Assertions have been autogenerated by utils/update_mlir_test_checks.py
+// RUN: circt-opt -create-dataflow -remove-block-structure %s | FileCheck %s
```

Comment:
Thanks for adding this.  It will probably be easier to maintain these tests if they are limited to just the -remove-block-structure pass and only have the operations handled by that pass.  (e.g. IR with blocks, and terminators).  What do you think about simplifying them?

Reply:
My thinking was if you have the same test case with&without -remove-block-structure, readers would be easier to understand the behavior of this pass. I can also simplify this though, both are fine for me.